Task_level_Parallelism/Data_driven/unique_task_regions/test.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 1986-2022 Xilinx, Inc. All Rights Reserved.
+ * Copyright 2022-2023 Advanced Micro Devices, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "test.h"
+
+void copy(hls::stream<int>& in, hls::stream<int>& out) {
+    for (int i = 0; i < 8; i++)
```

Comment:
hi,
isn't it supposed to be (induction variable bound to 16 x 4 times top func call = processing 64 elements):
`for (int i = 0; i < 16; i++)`

Reply:
Got it! Disregarded "two" calls in the top func body. Thus adds up